pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgadmin_types.go

```diff
@@ -52,6 +52,11 @@ type PGAdminPodSpec struct {
 	// +optional
 	Config PGAdminConfiguration `json:"config,omitempty"`
 
+	// A Secret containing the value for the LDAP_BIND_PASSWORD setting.
+	// More info: https://www.pgadmin.org/docs/pgadmin4/latest/ldap.html
+	// +optional
+	LDAPBindPassword *corev1.SecretKeySelector `json:"ldapBindPassword,omitempty"`
```

Comment:
🤔 I just noticed. Should this be in the `config` section; in the `PGAdminConfiguration` struct?

Reply:
:thinking: Originally, I was thinking we might want to keep this setting off on its own since it was more of a special case than the more general configuration settings, but I do think putting it in `PGAdminConfiguration` might make more sense. I'll work on that change.